docs/pages/database-access/guides/redshift-serverless.mdx

```diff
@@ -0,0 +1,400 @@
+---
+title: Database Access with Redshift Serverless on AWS
+description: How to configure Teleport Database Access with AWS Redshift Serverless.
+---
+
+This guide will help you to:
+
+- Set up Teleport to access your AWS Redshift Serverless instances.
+- Connect to your databases through Teleport.
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+![Teleport Database Access Redshift Self-Hosted](../../../img/database-access/guides/redshift_selfhosted_serverless.png)
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+![Teleport Database Access Redshift Cloud](../../../img/database-access/guides/redshift_cloud_serverless.png)
+</ScopedBlock>
+
+## Prerequisites
+
+- A pre-existing Teleport `(=teleport.version=)` cluster. If you don't already
+  have Teleport running, see [How to Choose a Teleport Edition](../../choose-an-edition/introduction.mdx)
+  to learn more about your options, and how to set up Teleport.
+- AWS account with a Redshift Serverless configuration and permissions to create
+  and attach IAM policies.
+- Command-line client `psql` installed and added to your system's `PATH` environment variable.
+- A host where you will run the Teleport Database Service. This guide assumes an
+  EC2 instance, and provides a corresponding example of access control.
+- (!docs/pages/includes/user-client-prereqs.mdx!)
+
+## Step 1/5. Create an IAM policy for Teleport
+
+<Admonition type="note" title="Example Access Control">
+This guide provides an example configuration of IAM access roles as a model,
+and uses an EC2 instance to serve the Teleport Database Access service.
+The level of access provided may not suit your needs, or may not fit your
+organization's access conventions. You should adjust the AWS IAM permissions to
+fit your needs.
+</Admonition>
+
+Teleport needs AWS IAM permissions to be able to:
+
+- Discover and register Redshift databases.
+- Manage IAM user or IAM role policies.
+
+If you don't already have one, create a role to apply to the EC2 instance that
+the Teleport Database Service will run on. This guide uses the example name
+`teleport-redshift-node`.
```

Comment:
nit
```suggestion
`teleport-redshift-serverless-node`.
```
I think it's better to have the serverless in the names to differentiate between the serverless vs the non-serverless. Same applies to other names.

Also, curious if we call these *node* or *server* or *host*?

Reply:
I wasn't sure myself to I reviewed some of our other docs and discussions, and it looks like *node* is still an appropriate term for a system running a Teleport access service but not the auth/proxy.

Did you also update the references to this renamed role, or should I go through and find/replace?

Reply:
> I wasn't sure myself to I reviewed some of our other docs and discussions, and it looks like node is still an appropriate term for a system running a Teleport access service but not the auth/proxy.

Cool. I was just curious.

> Did you also update the references to this renamed role, or should I go through and find/replace?

No. You may need to find all the `redshift` without serverless and replace them.

Reply:
Updated throughout in a3d1684df72a0b5a16b3d08ef993f286847c1c37